README.md

```diff
@@ -171,6 +173,7 @@ If you need some inspiration, [start with this](https://www.youtube.com/watch?v=
 - [The myth of the "Real JavaScript Developer"](https://www.youtube.com/watch?v=Xt5qpbiqw2g) - by _Brenna O'Brien_
 - [V8, modern JavaScript, and beyond](https://www.youtube.com/watch?v=N1swY14jiKc) - by _Seth Thompson_
 - [What the heck is the event loop anyway?](https://www.youtube.com/watch?v=8aGhZQkoFbQ) - by _Philip Roberts_
+- [All about React](https://www.reactjsvideos.com/#/search) - _Team Reactjsvideos_
```

Comment:
thanks @kaiiyer for your contribution. I am however only documenting presentations in conferences. Would you be willing to only including presentation talks? Thanks!